src/engines/arps/ARP/screen.hpp

```diff
@@ -12,50 +15,73 @@ namespace otto::engines::arp {
   inline std::string to_string(Playmode pm) noexcept
   {
     switch (pm) {
-    case Playmode::up: return "Up";
-    case Playmode::down: return "Down";
-    case Playmode::updown: return "Up/Down";
-    case Playmode::downup: return "Down/Up";
-    case Playmode::updowninc: return "Up/Down Inc.";
-    case Playmode::downupinc: return "Down/Up Inc.";
-    case Playmode::manual: return "Manual";
-    case Playmode::chord: return "Chord";
-    case Playmode::random: return "Random";
+      case Playmode::up: return "UP";
+      case Playmode::down: return "DOWN";
+      case Playmode::updown: return "UP/DOWN";
+      case Playmode::downup: return "DOWN/UP";
+      case Playmode::updowninc: return "UP/DOWN INC.";
+      case Playmode::downupinc: return "DOWN/UP INC.";
+      case Playmode::manual: return "MANUAL";
+      case Playmode::chord: return "CHORD";
+      case Playmode::random: return "RANDOM";
     };
     OTTO_UNREACHABLE;
   }
 
   inline std::string to_string(OctaveMode om) noexcept
```

Comment:
these functions used for formating strings for display should probably be called `display` or `to_display_string`
And they should definitely be declared/defined in the cpp file.

Also, they shouldn't return a `std::string`, but instead a `util::string_ref`

Reply:
Gotcha!